S3/English/ASS_32.ass

```diff
@@ -0,0 +1,111 @@
+Dialogue: 0,0:03:05.05,0:03:06.00,Default,,0,0,0,,{\pos(970,200)}Understand it
+Dialogue: 0,0:03:08.03,0:03:08.25,Default,,0,0,0,,{\c&H548AE5&\fnTrebuchet MS\pos(1635,805)}Being a part-time lifeguard
+Dialogue: 0,0:03:08.40,0:03:08.90,Default,,0,0,0,,{\c&H548AE5&\fnTrebuchet MS\pos(1635,805)}Being a part-time lifeguard
+Dialogue: 0,0:03:07.35,0:03:08.35,Default,,0,0,0,,Isn't that Neil
+Dialogue: 0,0:03:09.83,0:03:10.40,Default,,0,0,0,,Hold on
+Dialogue: 0,0:03:10.80,0:03:11.90,Default,,0,0,0,,Your brother is coming to rescue
+Dialogue: 0,0:03:11.90,0:03:13.30,Default,,0,0,0,,Stay away from me
+Dialogue: 0,0:03:14.00,0:03:14.55,Default,,0,0,0,,Don't be afraid, Neil
+Dialogue: 0,0:03:14.55,0:03:15.30,Default,,0,0,0,,Stay back
+Dialogue: 0,0:03:16.10,0:03:16.85,Default,,0,0,0,,I'm rescuing you
+Dialogue: 0,0:03:17.35,0:03:19.40,Default,,0,0,0,,Neil learns so fast
+Dialogue: 0,0:03:28.35,0:03:28.82,Default,,0,0,0,,Hello
+Dialogue: 0,0:03:29.25,0:03:30.90,Default,,0,0,0,,We are holding summer activity for lovers
+Dialogue: 0,0:03:31.57,0:03:32.64,Default,,0,0,0,,If you're alone...
+Dialogue: 0,0:03:32.78,0:03:33.94,Default,,0,0,0,,They aren't lovers at all
+Dialogue: 0,0:03:33.94,0:03:35.12,Default,,0,0,0,,Oh Lynn
+Dialogue: 0,0:03:35.31,0:03:36.61,Default,,0,0,0,,Why didn't you wait for me
+Dialogue: 0,0:03:37.95,0:03:39.03,Default,,0,0,0,,We're couple
+Dialogue: 0,0:03:39.12,0:03:40.08,Default,,0,0,0,,Could we come in
+Dialogue: 0,0:03:40.88,0:03:41.58,Default,,0,0,0,,Come in please
+Dialogue: 0,0:03:43.82,0:03:44.62,Default,,0,0,0,,Thanks for the help
+Dialogue: 0,0:03:45.15,0:03:45.90,Default,,0,0,0,,But I didn't mean to lie...
+Dialogue: 0,0:03:45.90,0:03:46.62,Default,,0,0,0,,Why are you so polite
+Dialogue: 0,0:03:47.25,0:03:48.00,Default,,0,0,0,,It's me
+Dialogue: 0,0:03:48.24,0:03:49.20,Default,,0,0,0,,Surprised?
+Dialogue: 0,0:03:50.65,0:03:52.61,Default,,0,0,0,,Transformation is an optional course in Devil University
+Dialogue: 0,0:03:52.94,0:03:54.81,Default,,0,0,0,,And Neil couldn't become such a beauty like me
+Dialogue: 0,0:03:55.35,0:03:56.55,Default,,0,0,0,,Are you shy?
+Dialogue: 0,0:03:57.90,0:03:59.20,Default,,0,0,0,,Just deceptive tricks
+Dialogue: 0,0:03:59.55,0:04:00.47,Default,,0,0,0,,Why would I be shy
+Dialogue: 0,0:04:00.98,0:04:01.88,Default,,0,0,0,,Who was deceptive
+Dialogue: 0,0:04:02.24,0:04:03.28,Default,,0,0,0,,Those are real
+Dialogue: 0,0:04:03.52,0:04:04.28,Default,,0,0,0,,Touch it if you don't belive me
+Dialogue: 0,0:04:05.21,0:04:06.00,Default,,0,0,0,,Pervert
+Dialogue: 0,0:04:06.60,0:04:08.00,Default,,0,0,0,,Why did you hurt me
+Dialogue: 0,0:04:14.38,0:04:15.64,Default,,0,0,0,,Abu
+Dialogue: 0,0:04:15.64,0:04:16.62,Default,,0,0,0,,We're leaving
+Dialogue: 0,0:04:17.18,0:04:18.44,Default,,0,0,0,,How long had he stay there
+Dialogue: 0,0:04:18.97,0:04:20.09,Default,,0,0,0,,Won't he become soak?
+Dialogue: 0,0:04:22.53,0:04:24.03,Default,,0,0,0,,But seems he really liked bathing
```

Comment:
"but it seems he really likes bathing"
"liked" work too but he's still bathing so I personally go with that being a constant fact than a one time one